glide.lock

```diff
@@ -68,7 +68,7 @@ imports:
 - name: github.com/uber-go/atomic
   version: e682c1008ac17bf26d2e4b5ad6cdd08520ed0b22
 - name: github.com/uber-go/tally
-  version: be9e53c77349ae2dd4b8c03a6dc20ed9a88b9927
+  version: 95078a8f10668bd1fa73ae46761cdc58d25436b8
```

Comment:
This is just a side effect of some glide update?

Reply:
We are using semver for tally so this is a new release.